nuxeo-ai-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-ai/admin/admin-ai-export.html

```diff
@@ -0,0 +1,169 @@
+<dom-module id="nuxeo-admin-ai-export">
+    <template>
+        <style>
+            #statsResult {
+                height: 60vh;
+            }
+
+            .row-container {
+                @apply --layout-horizontal;
+                @apply --layout-wrap;
+            }
+        </style>
+
+        <nuxeo-operation id="aiStats" op="AI.DatasetStats" response="{{stats}}"></nuxeo-operation>
+        <nuxeo-operation id="aiExport" op="AI.DatasetExport" ></nuxeo-operation>
+        <nuxeo-card heading="[[i18n('admin.ai.export')]]">
+            <paper-textarea id="queryInput"
+                            type="search"
+                            label="[[i18n('NXQL Query')]]"
+                            value="{{query}}"
+                            placeholder="[[i18n('imaging.query.placeholder')]]"
+                            autofocus>
+            </paper-textarea>
+            <paper-input id="inputs" label="[[i18n('admin.ai.inputs')]]" value="{{in_props}}"></paper-input>
+            <paper-input id="outputs" label="[[i18n('admin.ai.outputs')]]" value="{{out_props}}"></paper-input>
+            <paper-input id="split" label="[[i18n('admin.ai.split')]]" value="{{split_prop}}"></paper-input>
+            <paper-button on-tap="_recompute">[[i18n('admin.ai.stats.action')]]</paper-button>
+            <paper-button on-tap="_export">[[i18n('admin.ai.export.action')]]</paper-button>
+        </nuxeo-card>
+        <nuxeo-card heading="[[i18n('admin.ai.stats')]]">
+            <div class="row-container">
+                Total possible: [[_total.numericValue]] &nbsp;
+                Valid count: [[_count.numericValue]]
```

Comment:
You can use labels here. They can have placeholders for the parameters (`"my.label": "Stuff here: {0}",`) that follow the label on the i18n function.